### PR TITLE
implementation ShoppingCart and Ticket added

### PR DESCRIPTION
**Pull request by hpavlo**

### CODE REVIEW

**Review by YuriiHryniv — APPROVED** [review]
Looks great!

File: src/main/java/mate/academy/model/ShoppingCart.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "shopping_carts")
+public class ShoppingCart {
+    @Id
+    private Long id;
+    @OneToMany
+    private List<Ticket> tickets;
```
Comment (ChumakAn) [inline src/main/java/mate/academy/model/ShoppingCart.java:18]: let's add a @JoinColumn annotation here 
smth like    
 ```
@JoinTable(name = "",
            joinColumns = @JoinColumn(name = ""),
            inverseJoinColumns = @JoinColumn(name = ""))
```

File: src/main/java/mate/academy/model/Ticket.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "tickets")
+public class Ticket {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(fetch = FetchType.LAZY)
+    private MovieSession movieSession;
```
Comment (ChumakAn) [inline src/main/java/mate/academy/model/Ticket.java:18]: let's add a JoinColumn annotation here too

File: src/main/java/mate/academy/model/Ticket.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    @ManyToOne(fetch = FetchType.LAZY)
+    private User user;
```
Comment (ChumakAn) [inline src/main/java/mate/academy/model/Ticket.java:20]: let's add a JoinColumn annotation here too

File: src/main/java/mate/academy/service/impl/ShoppingCartServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    public ShoppingCart getByUser(User user) {
+        return shoppingCartDao.getByUser(user).orElseThrow(() ->
+                new EntityNotFoundException("Can't get shopping cart by user " + user));
+    }
+
+    @Override
+    public void registerNewShoppingCart(User user) {
+        ShoppingCart shoppingCart = new ShoppingCart();
+        shoppingCart.setUser(user);
+        shoppingCart.setTickets(new ArrayList<>());
```
Comment (ChumakAn) [inline src/main/java/mate/academy/service/impl/ShoppingCartServiceImpl.java:43]: it should work without setting the empty list

**Review by ChumakAn — CHANGES_REQUESTED** [review]
Good! Let's make it better

**Review by ChumakAn — APPROVED** [review]

**Pull request closed without merging**